clans/src/main/java/me/mykindos/betterpvp/clans/clans/ClanManager.java

```diff
@@ -177,6 +184,23 @@ public Optional<Clan> getClanByChunk(Chunk chunk) {
         return getClanById(uuid);
     }
 
+    public void applyDisbandClaimCooldown(ClanTerritory clanTerritory) {
+        setClaimCooldown(clanTerritory.getWorldChunk(), (long) (claimDisbandCooldown * 1000L));
+    }
```

Comment:
not really necessary coinsidering all it does is call another function

Reply:
I'm not sure if I want to also do things regular unclaiming, or add a way to designate an area unclaimable that admins can use (for whatever reason), which is why this was organized this way. Also, so that the config was defined in clanManager instead of the event listener.